src/classes/OPP_OpportunityContactRoles_TDTM.cls

```diff
@@ -188,10 +188,6 @@ public without sharing class OPP_OpportunityContactRoles_TDTM extends TDTM_Runna
                 //check if our contact lookup field is incorrect and update it
                 if (thisOpp.Primary_Contact__c != cr.ContactId) {
                     
-                    //update existing opp for OPP_OpportunityNaming call
```

Comment:
It was from before I rewrote the naming to just use contact roles. It didn't actually work and we have no way of testing the way the "New Contact Donation" button creates contact roles except manually. I made a story to make a selenium test for this button since we can't test it in apex.